frontend/src/components/AllPosts.js

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
 import PostsList from './PostsList'
 import { usePosts } from '../utils/customHooks/usePosts'
+import PostSkeletonTemplate from './PostSkeletonTemplate'
 
 const AllPosts = () => {
   const posts = usePosts()
 
   if (!posts) {
-    return null
+    return <PostSkeletonTemplate />
```

Comment:
What if there are no posts? Should we also display skeleton?

Reply:
If there are no posts after loading we should fallback to some default empty page -> no posts / empty category